networking/network_observability/configuring-operator.adoc

```diff
@@ -30,4 +30,8 @@ include::modules/network-observability-SRIOV-configuration.adoc[leveloffset=+1]
 For more information about creating the `SriovNetwork` custom resource, see xref:../../networking/hardware_networks/configuring-sriov-device.adoc#cnf-creating-an-additional-sriov-network-with-vrf-plug-in_configuring-sriov-device[Creating an additional SR-IOV network attachment with the CNI VRF plugin].
 
 include::modules/network-observability-resource-recommendations.adoc[leveloffset=+1]
-include::modules/network-observability-resources-table.adoc[leveloffset=+2]
\ No newline at end of file
+include::modules/network-observability-resources-table.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+For more information about troubleshooting memory consumption of the operator, see xref:../../networking/network_observability/troubleshooting-network-observability.adoc#controller-manager-pod-runs-out-of-memory_network-observability-troubleshooting[Network Observability controller manager pod runs out of memory].
```

Comment:
I don't remember exactly what we discussed, but in fact I think this is more in the Install operator section that it makes sense: when installing NOO (ie before we can even configure it), it may happen that we need to allocate more memory for it.
Maybe we could write something like "The operator actual memory consumption depends on your cluster size and the number of resources deployed. In might need to be adjusted accordingly, for more information refer to ...."  ?

Reply:
I saved the handy Slack link to this [covo](https://redhat-internal.slack.com/archives/C02939DP5L5/p1695650358697559?thread_ts=1695649320.998809&cid=C02939DP5L5) so I could remember to make this change :) . I actually don't think there was a super clear consensus on the best place, but I was thinking based on that convo that we should add the link to the Configuration area. But on second thought it seems better placed in the Installing area, so I can adjust it based on your comment.

Reply:
Can/should we also link to the resource considerations table? Maybe it would be a helpful reference in addition to the troubleshooting topic.

Reply:
I went ahead and added it but if you think its overkill, we can take it away. Seems like it could be helpful at this point in installation too.

Reply:
sounds good, thanks!